### ISSUE TITLE
Farm vs. restaurant visual distinction

### ISSUE THREAD

**Issue by zachlevitt** [community-table-market/addison-county#32]
**Is your feature request related to a need you have or imagine others having? Please describe.**
As a customer it would be helpful for there to be more visual distinction between farms and restaurants in terms of design of the menus, coloring, etc. I think it would be satisfying if it felt like a unique experience to shop from each type of organization. 

**Describe the solution you think would fulfill this need**
I think the landing pages for farms vs. restaurants should have different colors or header images to allow users to understand the distinction. These colors should be consistent when farms and restaurants are listed in the basket or the individual farm/restaurant pages. Overall, I think more visual distinction between each farm and restaurant would make the experience of using the site feel more customized and exciting. You could allow restaurant/farm owners to define the font/color/image/etc to make the styling more similar to the organization's marketing.

**Comment by justindoran** [comment]
Yes, this is a very, very good suggestion and something that @lindseyberk has talked about as well. We've just put up a revised restaurant interface that demonstrates our trajectory for this feature, but we really want the farm interface to feel very distinctive. Since restaurants are our focus for the next two releases, we'll get back to this in a few weeks and see what changes can be made. In the meantime, we would love to hear what you think a really cool farm menu and list interface would look like.
